LandSandBoat/server
# [SMN] Correct Titan animations & Blood Pact abilities

**_I affirm:_**

## What does this pull request do?



Updates all of Titan's animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Removes Titan's animation shifting from core
Added SQL updates for future Titan Blood Pact abilities

## Steps to test these changes


Use Titan's BP Ward to see proper MP consumption and messages displayed
Use Titan's BP Rage to see proper MP consumption and messages displayed
